examples/account-management.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import IBMRuntimeService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Examples\n",
+    "\n",
+    "## Account Management\n",
+    "\n",
+    "### Save account"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# save cloud account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    auth=\"cloud\", name=\"cloud-account-1\", token=\"123\", locator=\"Quantum-Service-34\"\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# save legacy account\n",
+    "IBMRuntimeService.save_account(\n",
+    "    auth=\"cloud\",\n",
+    "    name=\"legacy-account-1\",\n",
+    "    token=\"456\",\n",
+    "    hub=\"ibm-q\",\n",
+    "    group=\"open\",\n",
+    "    project=\"main\",\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### List saved accounts"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# list all accounts saved on disk\n",
+    "IBMRuntimeService.saved_accounts()"
```

Comment:
@rathishcholarajan, @jyu00 - what do you think would users like as a return value here?

Previously a simple dict was returned.
```
{"token": credentials.token, "url": credentials.url}
```

With the new account management functionality we have to decide what to return instead. I propose to return the a `dict` that gives access to the configuration as saved on disk (https://github.com/Qiskit/qiskit-ibm-runtime/issues/31#issue-1066369790). 
![image](https://user-images.githubusercontent.com/16404496/145406260-657552e1-302e-4a0f-882b-db2082e6dcf2.png)